examples/MultiXcodeProject/Coffee/Coffee.podspec

```diff
@@ -0,0 +1,11 @@
+Pod::Spec.new do |spec|
```

Comment:
dir should be `examples/MultiplePodsProjects Example` for consistency with the other exampels